app/components/insert-title-card.js

```diff
@@ -0,0 +1,21 @@
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import insertTitle from '../commands/insert-title';
+import { inject as service } from '@ember/service';
+
+export default class InsertTitleCardComponent extends Component {
+  @service intl;
+
+  @action
+  insertTitle() {
+    this.args.controller.doCommand(insertTitle(this.intl), {
```

Comment:
rather than passing intl, I'd stick to the pattern we have for other plugins and just pass in the actual string.
and preferably with named arguments, so:
```js
this.args.controller.doCommand(insertTitle({placeholder: this.intl.t('reglement-edit.document-title-placeholder')}))
```